test/testpreprocessor.cpp

```diff
@@ -48,18 +47,30 @@ class TestPreprocessor : public TestFixture {
     TestPreprocessor() : TestFixture("TestPreprocessor") {}
 
 private:
-    static std::string expandMacros(const char code[], ErrorLogger &errorLogger) {
-        std::istringstream istr(code);
+    template<size_t size>
```

Comment:
I don't see the advantage at all here neither. What exactly is the advantage?

Reply:
We do not need the stream and get the size.